Apps/W1/Shopify/app/src/GraphQL/Codeunits/ShpfyGQLDraftOrderComplete.Codeunit.al

```diff
@@ -0,0 +1,27 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy GQL DraftOrderComplete (ID 30312) implements Interface Shpfy IGraphQL.
+/// </summary>
+codeunit 30312 "Shpfy GQL DraftOrderComplete" implements "Shpfy IGraphQL"
+{
+    Access = Internal;
+
+    /// <summary>
+    /// GetGraphQL.
+    /// </summary>
+    /// <returns>Return value of type Text.</returns>
+    internal procedure GetGraphQL(): Text
+    begin
+        exit('{"query": "mutation {draftOrderComplete(id: \"gid://shopify/DraftOrder/{{DraftOrderId}}\") { draftOrder { order { id, name, fulfillmentOrders(first: {{NumberOfOrders}}) {nodes{ id, status}} } id, status} userErrors { field, message }}}"}');
+    end;
+
+    /// <summary>
+    /// GetExpectedCost.
+    /// </summary>
+    /// <returns>Return value of type Integer.</returns>
+    internal procedure GetExpectedCost(): Integer
+    begin
+        exit(10);
```

Comment:
Did you check the cost?

When you execute a query, in the return message you can see costs, and I just use a "rounded" number of what I see there.

Reply:
That's one of the questions I wanted to ask you in our standup meeting, should I just use that number from the Postman.